backend/penndata/views.py

```diff
@@ -144,6 +146,10 @@ def get_queryset(self):
         return HomePageOrder.objects.all()
 
 
+# Tracks the time user opens up Penn Mobile homepage
+@LabsAnalytics.record_apiview(
+    ViewEntry(name="homepage_visit_time", get_value=lambda req, res: timezone.localtime())
```

Comment:
^ same as above, can maybe just store a "1" or something and we can SQL query this info later.